Tipstaff/Controllers/ChildAbductionController.cs

```diff
@@ -106,10 +106,10 @@ public ViewResult Index(ChildAbductionListViewModel model)
                     TRs = TRs.OrderByDescending(a => a.Respondents.Count()).ThenBy(b => b.tipstaffRecordID);
                     break;
                 case "uniqueid asc":
-                    TRs = TRs.OrderBy(a => a.tipstaffRecordID).ThenBy(b => b.tipstaffRecordID);
+                    TRs = TRs.OrderBy(b => int.Parse(b.tipstaffRecordID));
```

Comment:
can you please try to parse once  and use it whenever u need? It's a case method but still there is a penalty when this code is converted into IL first time..